docs/tests/e2e-suite-desktop.md

```diff
@@ -0,0 +1,20 @@
+# @trezor/suite-desktop e2e tests
+
+**currently works only on Linux, volunteers to make it work on Mac needed**
```

Comment:
I am pretty sure I was able to work like this on linux when I was creating this test but @vdovhanych doen't think it is possible. cc @ondracja

Reply:
maybe he is right, maybe it is because network_mode:host in trezor-user-env-unix setup

Reply:
@mroz22 yes, that's why it was working on Linux, as you were not setting up the ports but instead using the host network.